qi-doc/scribblings/interface.scrbl

```diff
@@ -352,6 +352,8 @@ The first and most common way is to simply wrap the expression with a @racket[ge
 
 The second way is if you want to describe a @tech{flow} using the host language instead of Qi. In this case, use the @racket[esc] form. The wrapped expression in this case @emph{must} evaluate to a function, since functions are the only values describable in the host language that can be treated as flows. Note that use of @racket[esc] is unnecessary for function identifiers since these are @seclink["Identifiers"]{usable as flows directly}, and these can even be @seclink["Templates_and_Partial_Application"]{partially applied using standard application syntax}, optionally with @racket[_] and @racket[___] to indicate argument placement. But you may still need @racket[esc] in the specific case where the identifier collides with a Qi form.
 
+Note that if you use @racket[esc] to define your flow, the @seclink["It_s_Languages_All_the_Way_Down"]{Qi compiler} will not attempt to optimize it. If such a flow occurs within a larger flow, other parts of that containing flow would still be optimized as usual, but as escaped forms are generally not considered in optimizations, there may be few applicable nonlocal optimizations (i.e. those over more than one flow, including for @seclink["Phrases"]{phrases}) involving the escaped flow.
```

Comment:
It's in the sense of "there are few who would volunteer for such a dangerous mission," i.e. implying that there aren't many, rather than the positive that, indeed, there _are_ a few, a more hopeful message. If that makes sense? The intent is to provide a warning against use of `esc` in this context. Another phrasing could be:

```
but as escaped forms are generally not considered in optimizations, there may not be many applicable nonlocal optimizations
```

... but this has many negatives "not considered" "not many" "nonlocal." I think I prefer the existing phrasing between these two alternatives, but maybe there is an even clearer phrasing we can think of.

Reply:
The sentence is getting long, the more I look at it. A clean break around "but" would help: "If X occurs, then Y. Since Z is not considered, then A."